components/blitz/src/omero/gateway/model/ROIData.java

```diff
@@ -209,9 +209,11 @@ public void removeShapeData(ShapeData shape)
         ROICoordinate coord = shape.getROICoordinate();
         List<ShapeData> shapeList;
         shapeList = roiShapes.get(coord);
```

Comment:
`List<ShapeData> shapeList = roiShapes.get(coord);` in one line would look better